src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -28,9 +28,6 @@ public static IEnumerable<object[]> Match_Basic_TestData()
             // Testing control character escapes???: "2", "(\u0032)"
             yield return new object[] { "(\u0034)", "4", RegexOptions.None, 0, 1, true, "4", };
 
-            // Using *, +, ?, {}: Actual - "a+\\.?b*\\.?c{2}"
-            yield return new object[] { @"a+\.?b*\.+c{2}", "ab.cc", RegexOptions.None, 0, 5, true, "ab.cc" };
```

Comment:
Where is this one duplicated?

Reply:
https://github.com/dotnet/runtime/blob/5e6711dbfb1b696e1fb7c53f9d84d33c60dc87be/src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs#L169